tests/baselines/reference/emptyThenWarning.errors.txt

```diff
@@ -0,0 +1,15 @@
+tests/cases/compiler/emptyThenWarning.ts(1,6): warning TS20000: Potentially wrong empty statement in THEN clause.
+tests/cases/compiler/emptyThenWarning.ts(4,19): warning TS20000: Potentially wrong empty statement in THEN clause.
+
+
+==== tests/cases/compiler/emptyThenWarning.ts (2 errors) ====
+    if(1);
+         ~
+!!! warning TS20000: Potentially wrong empty statement in THEN clause.
+    
+    let x = 0;
+    if (true === true); {  
+                      ~
+!!! warning TS20000: Potentially wrong empty statement in THEN clause.
+        x = 1;
+    }
```

Comment:
Can you add a test that shows that

```ts
if(someComplexConditionICantBeBotheredToInvert) {
}
else {
    // some code ...
}
```

does *not* warn? It's not good style, but it's intended, so I want to document that it will not warn.